poolrpc/trader.proto

```diff
@@ -1011,6 +1018,42 @@ message RecoverAccountsResponse {
     uint32 num_recovered_accounts = 1;
 }
 
+message AccountModificationFeesRequest {
+}
+
+message AccountModificationFee {
+    // Modification action type.
+    string action = 1;
+
+    // Action transaction block height.
+    int32 block_height = 2;
+
+    // Action transaction timestamp.
+    int64 timestamp = 3;
+
+    // Action transaction output amount.
+    int64 output_amount = 4;
```

Comment:
I think it would be great to also have the TXID (as a string, otherwise we have the byte reverse order problem) and the amount diff as well. Then we would be able to correlate it to a wallet transaction in the UI if we wanted/needed to. Can be a follow-up PR to unblock this.

Reply:
Locally, I've added the `txid`. Will push an update.

Isn't amount diff redundant in that a consumer could just calculate it from the output amounts the RPC already provides? I don't think the RPC should provide a different view (diffs) onto the same data. Unless it's something that would be needed very commonly I suppose?